samples/apps/fireworks-studio/.virtual_documents/notebooks/tutorial.ipynb

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
what is this file?!

Reply:
I don't really know why it existed - seemed like somebody checked in accidentally.